front/src/styles/scss/applications/operationalStudies/_waypointsPanel.scss

```diff
@@ -0,0 +1,169 @@
+.waypoints-panel {
+  margin: 0;
+  width: 28.5rem; // TODO switch width to 44.5rem (712px) after implementing styles columns
+  height: 100vh;
+  max-height: 100vh;
+  font-family: 'IBM Plex Sans', sans-serif;
+
+  &::backdrop {
+    background: var(--modal-osrd-overlay);
+  }
+
+  .waypoints-panel-header {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 3.25rem;
+    font-size: 0.875rem;
+    color: var(--grey50);
+    background-color: var(--ambiantB15);
+
+    &::after {
+      content: '';
+      position: absolute;
+      bottom: 0.25rem;
+      left: 0;
+      width: 100%;
+      height: 0.0625rem;
+      background-color: var(--grey30);
+    }
```

Comment:
Because on the mockup, this grey line is not at the bottom of the header but 4px above :) (that's why I put a `bottom: 0.25rem`)

![Capture d’écran 2024-10-17 à 15 03 38](https://github.com/user-attachments/assets/517c1a9a-7c3d-4476-a11a-7eaf0ae402fa)